php/sync/class-upload-sync.php

```diff
@@ -136,18 +136,35 @@ public function add_inline_action( $actions, $post ) {
 					esc_attr__( 'Sync and deliver from Cloudinary', 'cloudinary' ),
 					esc_html__( 'Sync and deliver from Cloudinary', 'cloudinary' )
 				);
-			} else {
-				if ( file_exists( get_attached_file( $post->ID ) ) ) {
-					$actions['cloudinary-push'] = sprintf(
-						'<a href="%s" aria-label="%s">%s</a>',
-						$action_url,
-						esc_attr__( 'Re-sync to Cloudinary', 'cloudinary' ),
-						esc_html__( 'Re-sync to Cloudinary', 'cloudinary' )
-					);
-				}
+			} else if ( file_exists( get_attached_file( $post->ID ) ) ) {
+				$actions['cloudinary-push'] = sprintf(
+					'<a href="%s" aria-label="%s">%s</a>',
+					$action_url,
+					esc_attr__( 'Re-sync to Cloudinary', 'cloudinary' ),
+					esc_html__( 'Re-sync to Cloudinary', 'cloudinary' )
+				);
 			}
 		}
 
+		if (
+			$this->sync->is_syncable( $post->ID )
+			&& ! empty( get_post_meta( $post->ID, Sync::META_KEYS['sync_error'], true ) )
+		) {
+			$actions['cloudinary-clean-up'] = sprintf(
+				'<a href="%s" aria-label="%s">%s</a>',
+				add_query_arg(
+					array(
+						'action'   => 'cloudinary-clean-up',
+						'media[]'  => $post->ID,
+						'_wpnonce' => wp_create_nonce( 'bulk-media' ),
+					),
+					admin_url( 'upload.php' )
+				),
+				esc_attr__( 'Clean Cloudinary error', 'cloudinary' ),
+				esc_html__( 'Clean Cloudinary error', 'cloudinary' )
```

Comment:
<!-- trivial preference-based requests -->
**🥜 nitpick**
Same here, maybe capitalise Error to keep it consistent

Reply:
```suggestion
				esc_html__( 'Clean Cloudinary Error', 'cloudinary' )
```